### ISSUE TITLE
Incompatible for CLion 2024.2.1

### ISSUE THREAD

**Issue by vuthanhtrung2010** [Pushpavel/AutoCp#133]
![image](https://github.com/user-attachments/assets/158bcddc-5ed2-4417-b507-ea6b306dfebd)

The issue started when I just updated my CLion

**Comment by znzryb** [comment]
That's right. The plugin is not compatible with 2024.2.1 since 0.8.0 (perhaps). Just update clion to 2024.3 or later version.
